src/main/java/hh/forest_of_habits/service/impl/ForestServiceImpl.java

```diff
@@ -97,4 +98,33 @@ private List<TreeResponse> sort(List<TreeResponse> trees) {
         });
         return result;
     }
+
+    @Override
+    public StatResponse getStat() {
+        List<ForestResponse> forests = mapper.mapAll(forestRepository.findByUser_name(AuthFacade.getUsername()));
```

Comment:
```suggestion
       var forests = forestRepository.findByUser_name(AuthFacade.getUsername());
```
Response вроде не нужен, дальше деревья еще раз получаются из запроса